### PR TITLE
Updated numFOCUS language

### PR DESCRIPTION
**Pull request by bhuvanakundumani**
Updated numFOCUS language in donate.rst .

### CODE REVIEW

File: donate.rst [hunk 1]
```diff
@@ -2,9 +2,9 @@
 Donations
 *********
 
-The *pandas* Project will always be 100% open source software, free for all to use and released under the liberal terms of the modified BSD license. If you have found *pandas* to be useful in your work, research or company, please consider making a donation to the project commensurate with your resources. Any amount helps! All donations will be used strictly to fund the development of the *pandas* Projects's open source software, documentation and community.
```
Comment (jorisvandenbossche) [inline donate.rst]: I think we can keep this paragraph in some form (it is more about pandas than about NumFOCUS)

**Comment by jorisvandenbossche** [comment]
> I see that in the home page we also have a mention to NumFOCUS, do you think we should repeat it, or do you have a better idea?

Personally, I would find it better to keep the text on the homepage as is, if that is OK for NumFOCUS. There, we don't really explain what NumFOCUS is, and we link to the donate page where we then use the exact required text to reference NumFOCUS. 
On the home page (or at least the current home page as is), I don't find it fitting in the page to add the full paragraph where currently NumFOCUS is mentioned (although we do have the logo there, and they asked to update the text alongside the logo).

**Comment by Dr-G** [comment]
The homepage text looks fine; if we can update the donate page text, that's where it's most relevant.

Also of **high** importance: the donate page still uses the old (soon to be closed down) donation page. It should be: https://numfocus.salsalabs.org/donate-to-pandas

Or if you'd like the form widget:

`<div id="c3a7d378-061d-4979-b13e-430002d56452">
    <script type="text/javascript" src="https://default.salsalabs.org/api/widget/template/4ba4e328-1855-47c8-9a89-63e4757d2151/?tId=c3a7d378-061d-4979-b13e-430002d56452" ></script>
</div>`

**Comment by TomAugspurger** [comment]
@Dr-G FYI, I needed to change the widget code a bit.

```
   <div id="c3a7d378-061d-4979-b13e-430002d56452">
     <script type="text/javascript" src="https://default.salsalabs.org/api/widget/template/4ba4e328-1855-47c8-9a89-63e4757d2151/?tId=c3a7d378-061d-4979-b13e-430002d56452"></script>
   </div>
```

That `<div id="">` stuff is needed too, so that the javascript has a target to stuff the form into.

@bhuvanakundumani let's keep this focused on the text. I'll update the form in a different PR. I think @jorisvandenbossche had a comment https://github.com/pandas-dev/pandas-website/pull/79#discussion_r312888421 that may need addressing.

File: donate.rst (hunk 1 above)
Comment (bhuvanakundumani) [reply donate.rst]: @jorisvandenbossche have added the info on pandas in the donate page.

**Comment by bhuvanakundumani** [comment]
@Dr-G , @TomAugspurger Should i change the website link <https://www.numfocus.org/> to https://numfocus.salsalabs.org/donate-to-pandas in the donate page?  

Or we could add a line 'Please visit https://numfocus.salsalabs.org/donate-to-pandas to donate !! below the text about numFOCUS.

**Comment by TomAugspurger** [comment]
I think it's fine to keep the link to the main NumFOCUS site for general information. We have a direct link to the donation & the form below.

File: donate.rst [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
+*pandas* Project will always be 100% open source software, free for all to use and released under the liberal terms of the modified BSD license. If you have found *pandas* to be useful in your work, research or company, please consider making a donation to the project commensurate with your resources. Any amount helps! All donations will be used strictly to fund the development of the *pandas* Projects's open source software, documentation and community.
```
Comment (TomAugspurger) [inline donate.rst]: Can you add back the "The" here?

```suggestion
The *pandas* Project will always be 100% open source software, free for all to use and released under the liberal terms of the modified BSD license. If you have found *pandas* to be useful in your work, research or company, please consider making a donation to the project commensurate with your resources. Any amount helps! All donations will be used strictly to fund the development of the *pandas* Projects's open source software, documentation and community.
```

Comment (bhuvanakundumani) [reply donate.rst]: Have added ' The' back.

**Comment by TomAugspurger** [comment]
Thanks!

**Pull request merged**